apps/allocations/app/components/Panel/NewAccount.js

```diff
@@ -28,6 +27,7 @@ class NewAccount extends React.Component {
   createAccount = () => {
     const { description, } = this.state
     if (isStringEmpty(description)) {
+      // eslint-disable-next-line no-console
       return console.info(
```

Comment:
I think this is an error message; maybe we can use `console.error` here. I think our rules may be configured to allow that, so we'd be able to get rid of the `eslint-disable-next-line` declaration.

Reply:
Yes. But also.. this is actually very valid error message for the user. I made it show up in the panel, where it belongs.